matrix-sdk-android/src/main/java/org/matrix/android/sdk/api/session/events/model/Event.kt

```diff
@@ -498,6 +498,8 @@ fun Event.getPollContent(): MessagePollContent? {
     return getClearContent().toModel<MessagePollContent>()
 }
 
+fun Event.isJitsiEvent() = this.getClearType() == EventType.STATE_ROOM_WIDGET_LEGACY
```

Comment:
This looks too generic to me, won't that trigger for all kinds of widgets, not just jitsi?

Reply:
Updated detecting function to more accurate one